addon/ng2/models/webpack-build-common.ts

```diff
@@ -0,0 +1,127 @@
+import * as path from 'path';
```

Comment:
This file seems a remnant of a rebase, can you remove it? It's making the CI break.